shared/src/test/scala/squants/market/MoneySpec.scala

```diff
@@ -151,6 +151,12 @@ class MoneySpec extends FlatSpec with Matchers {
     x != y should be(right = true)
   }
 
+  it should "return consistent hashcode" in {
+    val someMoney = USD(2.1)
+
+    someMoney.hashCode() shouldBe someMoney.hashCode()
```

Comment:
Java requires equal hash codes for equal objects, so I suggest adding a test which also checks against another instance of `USD(2.1)`